.github/workflows/reusable-build.yml

```diff
@@ -213,7 +213,7 @@ jobs:
     strategy:
       fail-fast: false # Build and test everything so we can look at all the errors
       matrix:
-        node: [14, 16, 18]
+        node: [16, 18]
```

Comment:
Should we add an `engines` filed to the @rspack/core package.json

Reply:
good point